template.yaml

```diff
@@ -1,40 +1,37 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Description: >
-  Hello World lambda
-
-Metadata:
-  AWS::ServerlessRepo::Application:
-    Name: "lambda-template"
-    Description: "A GH template for quickly starting a new AWS lambda project."
-    Author: "Sage-Bionetworks"
-    SpdxLicenseId: "Apache-2.0"
-    # paths are relative to .aws-sam/build directory
-    LicenseUrl: "LICENSE"
-    ReadmeUrl: "README.md"
-    Labels: ["serverless", "template", "github", "quick-start"]
-    HomePageUrl: "https://github.com/Sage-Bionetworks-IT/lambda-template"
-    SemanticVersion: "0.0.3"
-    SourceCodeUrl: "https://github.com/Sage-Bionetworks-IT/lambda-template/tree/0.0.3"
+  MIPS transform and cache API
 
 # More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
   Function:
-    Timeout: 30
+    Timeout: 10
+
+Parameters:
+  SsmKeyAdminArns:
+    Type: CommaDelimitedList
+    Description: List of role ARNs to manage the encryption key
+  SsmAliasPrefix:
+    Type: String
+    Default: lambda-mips-api
 
 Resources:
-  HelloWorldFunction:
+  Function:
     Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
     Properties:
       CodeUri: .
-      Handler: hello_world/app.lambda_handler
+      Handler: mips_api.lambda_handler
       Runtime: python3.9
+      Environment:
+        Variables:
+          SsmPath: '/lambda/mipsSecret'
       Role: !GetAtt FunctionRole.Arn
       Events:
-        HelloWorld:
+        ServiceCatalogProgramCodes:
           Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
           Properties:
-            Path: /hello
+            Path: /all/accounts.json
```

Comment:
I called this 'accounts' because the mips term for the data is a 'chart of accounts' but I don't have a strong preference on the name, 'costcenters' works just as well.